turtlebot3_random_walk:
  ros__parameters:
    view_angle: 30.0
    min_rand: 10
    max_rand: 20
    safety_distance: 0.5
    linear_speed: 0.15
    angular_speed: 0.25
    update_rate: 0.5